extension/parquet/include/parquet.json

```diff
@@ -112,13 +112,7 @@
     "includes": [
       "parquet_writer.hpp"
     ],
-    "members": [
-      {
-        "id": 100,
-        "name": "ids",
-        "type": "unique_ptr<case_insensitive_map_t<FieldID>>"
-      }
-    ],
+    "custom_implementation": true,
```

Comment:
Can this not use the `"serialize_property": "ids.operator*()"`, and `deserialize_property` to avoid having to use a custom implementation?

Reply:
Done